docs/docs/caching.md

```diff
@@ -36,3 +42,7 @@ The following plugins have been created:
 - [gatsby-plugin-s3](https://github.com/jariz/gatsby-plugin-s3)
 
 When deploying with Now, follow the instructions in the [Now documentation](https://zeit.co/guides/deploying-gatsby-with-now#bonus:-cache-your-gatsby-assets).
+
+---
+
+<sup>1</sup> It's important that you use the combination of `max-age=0` and `must-revalidate` instead of using `no-cache`. This allows the CDN to store copies of the files on the servers closest to your users and only download a new version from the origin server in the event that the files have changed. Using `no-cache`, on the other hand, decreases performance because it forces the CDN to download a new copy of the file from the origin server on every request.
```

Comment:
This may be too in the weeds; however, it also would have been very useful for me to read when I first saw this page.

Reply:
I'm not experienced with CDNs, but this info in the docs would definitely made things clearer for me 👍